src/Xamarin.Android.Build.Tasks/Tasks/ResolveSdksTask.cs

```diff
@@ -314,6 +322,48 @@ public bool RunTask ()
 			return !Log.HasLoggedErrors;
 		}
 
+		static readonly Regex javaVersionRegex = new Regex (@"""([\d\.]+)_\d+""");
+
+		void ValidateJavaVersion (string minRequiredJavaVersion)
+		{
+			Version minimumRequiredJavaVersion;
+			if (!Version.TryParse (minRequiredJavaVersion, out minimumRequiredJavaVersion)) {
+				Log.LogError ($"Failed to parse the MinimumRequiredJavaVersion value {minRequiredJavaVersion}. Please make sure this is a valid version number.");
+				return;
+			}
+
+			var proc = new Process ();
+			proc.StartInfo.FileName = Path.Combine (JavaSdkPath, "bin", OS.IsWindows ? "java.exe" : "java");
+			proc.StartInfo.Arguments = "-version";
+			proc.StartInfo.CreateNoWindow = true;
+			proc.StartInfo.RedirectStandardOutput = true;
+			proc.StartInfo.RedirectStandardError = true;
+			proc.StartInfo.UseShellExecute = false;
+
+			try {
+				proc.Start ();
+				proc.WaitForExit ();
+			} catch (Exception ex) {
+				Log.LogWarningFromException (ex);
+				Log.LogWarning ($"Failed to get the Java SDK version. Please ensure you have Java {minRequiredJavaVersion} or above installed.");
+				return;
```

Comment:
This doesn't `Dispose()` of the Process instance. You should wrap the `new Process()` in a `using` block to ensure we don't keep a process handle open longer than necessary.